config/environments/staging.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
```

Comment:
Suggestion: There's a lot of settings here that we probably want to keep the same between staging/prod. Can we just include the prod file like we do in https://github.com/pulibrary/figgy/blob/db_lando/config/environments/staging.rb?